components/utils/highlighter.js

```diff
@@ -0,0 +1,13 @@
+export default highlighter
+
+function highlighter(highlight) {
```

Comment:
Yeah, let's simplify this. Could we do a single function that takes all the args? Something like:

```
function highlighter(highlight, language, options, value) {
  const code = highlight(value, language)
  return {component, code, ...options}
}
```

And we could inline it directly in the original file?